attestation-service/Cargo.toml

```diff
@@ -50,6 +52,9 @@ tokio.workspace = true
 tonic = { workspace = true, optional = true }
 uuid = { version = "1.1.2", features = ["v4"] }
 x509-parser = { version = "0.14.0", optional = true }
+# TODO: change it to "0.1", once released.                                                                                                                                         
+csv-rs = { git = "https://gitee.com/anolis/csv-rs", rev = "8a90aac", optional = true }
+codicon = { version = "3.0", optional = true }
```

Comment:
Hey @BaoshunFang Could you please move the two deps in a dictionarial order?

Reply:
Done